app/src/main/java/com/mobile/tandil/kotlinbaseproject/database/ReservationDataBase.kt

```diff
@@ -0,0 +1,17 @@
+package com.mobile.tandil.kotlinbaseproject.database
+
+import com.mobile.tandil.kotlinbaseproject.entity.Reservation
+
+class ReservationDataBase {
+    private var reservations: HashMap<Int, List<Reservation>> = hashMapOf()
```

Comment:
this is not fine, you should access this hashmap through a singleton pattern to always use the same hashmap. Create a getInstance method that make this